packages/server/src/synchronizer/check-duplicate-pages.ts

```diff
@@ -0,0 +1,23 @@
+import chalk from 'chalk'
+
+export const checkDuplicatePages = (entries: string[]) => {
+  const allPages = entries.filter((page) => page.includes('pages'))
```

Comment:
One last thing hope you don't mind. This one looks a lot like the other one. Any chance we can create a shared function?

Reply:
Good idea, done!